<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XXXXClass</class>
 <widget class="QMainWindow" name="XXXXClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XXXX</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="recvLabel">
            <property name="text">
             <string>RECV</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="m_recvTextEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="sendLabel">
            <property name="text">
             <string>SEND</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="m_sendTextEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="m_comboBox"/>
        </item>
        <item>
         <widget class="QComboBox" name="m_itemComboBox"/>
        </item>
        <item>
         <widget class="QComboBox" name="m_parityComboBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="m_pushButton">
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="m_timerCheckBox">
          <property name="text">
           <string>定时发送</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="xxxx.qrc"/>
 </resources>
 <connections/>
</ui>
